README.md

```diff
@@ -30,6 +33,9 @@ docker run --net=host -e DATA_SOURCE_NAME="postgresql://postgres:password@localh
 * `disable-settings-metrics`
   Use the flag if you don't want to scrape `pg_settings`.
 
+* `auto-discover-databases`
+  Whether to discover the databases on a server dynamically.
```

Comment:
Wheter? If we use wether the phrase would be `wether blah blah blah or not`.
Can we change it to something like `enable discovering databases automatically`?